src/Plugin/Visualisation/Style/TableVisualisationStyleBase.php

```diff
@@ -41,9 +41,12 @@ protected function buildTable(array $records) {
    *   An array of table build settings.
    */
   protected function tableBuildSettings(array $records) {
+    $config = $this->getConfiguration();
+
     return [
       'data' => $this->getTableRows($records),
       'columns' => $this->getTableHeader(),
+      'disable_table' => $config['chart']['disable_table'],
```

Comment:
Should be camel case here right for the js plugin?

Reply:
@rob-duplock are all the other configs camel case? camel is prefered but if everything else uses underscores then IMO consistency is more important

Reply:
In the interest in consistency with other parts of the module, I've now implemented this as `this.options.table.disable` / `$config['chart']['table']['disable']` which is how its done elsewhere.

Reply:
Hey @jez500 could you please give this PR another look? I have implemented fixes as discussed.